datasets/panstarrs.yaml

```diff
@@ -0,0 +1,24 @@
+Name: Pan-STARRS PS1 Survey
+Description: |
+  The PS1 surveys used a 1.8 meter telescope and its 1.4 Gigapixel camera to image the sky in five broadband filters. The largest of these surveys provides coverage of the entire sky north of -30 degrees declination.
+Documentation: https://outerspace.stsci.edu/display/PANSTARRS/
+Contact: archive@stsci.edu
+ManagedBy: "[Space Telescope Science Institute](http://www.stsci.edu/)"
+Citation: Please see the documentation for full citation instructions.
+UpdateFrequency: Never
+Tags:
+  - astronomy
+License: STScI hereby grants the non-exclusive, royalty-free, non-transferable, worldwide right and license to use, reproduce, and publicly display in all media data from the PS1 surveys.
```

Comment:
The license stuff is weird to me too.  Back in 2017 we made this statement about the copyright:

> These images are copyrighted by the University of Hawaii and are free for non-commercial use.  Hawaii is currently discussing the precise copyright statement for commercial use.  I believe that the images will be free for all to use but don't know that for certain yet.  We will be adding a statement to our site once that is settled.

We never did get a response from Hawaii on the details of their preferred license.  But they have approved the move of the data to the AWS public data, so the license needs to be whatever is appropriate for AWS.  We could try again to get a definitive answer from Hawaii on this, or we can take their approval of the AWS data availability as also approving the necessary license.

Reply:
Uh, ok. I guess if Hawaii were concerned about this, they would have followed up. Right now, the data is available in AWS with no license listed, so let's go for option #2.